pyproject.toml

```diff
@@ -0,0 +1,37 @@
+# License and classifier list:
+# https://pypi.org/pypi?%3Aaction=list_classifiers
+
+[build-system]
+requires = ["setuptools"]
+build-backend = "setuptools.build_meta"
+
+[project]
+authors = [
+    {name = "NASA EOSDIS Harmony Team", email = "christopher.d.durbin@nasa.gov"}
+]
+classifiers = [
+  "Programming Language :: Python",
+  "Programming Language :: Python :: 3",
+  "Operating System :: OS Independent",
+]
+description = "A library for Python-based Harmony services to parse incoming message, fetch data, stage data, and call back to Harmony."
+dynamic = ["dependencies", "optional-dependencies", "version"]
+license = { text = "License :: OSI Approved :: Apache Software License" }
+name = "harmony-service-lib"
+readme = "README.md"
+requires-python = ">= 3.8"
+
+[project.scripts]
```

Comment:
Per [this documentation](https://packaging.python.org/en/latest/specifications/pyproject-toml/#entry-points) `project.scripts` is equivalent to `console_scripts`.